Sources/Workspace/Workspace.swift

```diff
@@ -1067,10 +1068,13 @@ extension Workspace {
     /// Computes states of the packages based on last stored state.
     fileprivate func computePackageStateChanges(
         resolvedDependencies: [(RepositorySpecifier, BoundVersion)],
-        updateBranches: Bool
-    ) throws -> [RepositorySpecifier: PackageStateChange] {
+        updateBranches: Bool,
+        diagnostics: DiagnosticsEngine
+    ) -> [RepositorySpecifier: PackageStateChange] {
```

Comment:
Lets not convert this method because this is one of those methods which probably shouldn't return an invalid object.